utils/migration-scripts/src/sumer-giza/BaseMigration.ts

```diff
@@ -109,11 +109,23 @@ export abstract class BaseMigration {
   private async loadSudoKey() {
     const { sudoUri } = this.config
     const keyring = new Keyring({ type: 'sr25519' })
-    this.sudo = keyring.createFromUri(sudoUri)
+    const key = keyring.createFromUri(sudoUri)
+    if (!key) {
+      throw new Error(`Invalid key!`)
+    }
```

Comment:
`createFromUri` either returns a non-empty `KeyringPair` or throws, so no additional check should be needed
```suggestion
    const key = keyring.createFromUri(sudoUri)
```